src/vt-tv/render/render.cc

```diff
@@ -248,17 +265,11 @@ std::pair<double, double> Render::computeRankQoiRange_() {
 double Render::computeRankQoiAverage_(PhaseType phase, std::string qoi) {
   // Initialize rank QOI range attributes
   double rq_sum = 0.0;
-
-  if (qoi == "load"){
-    auto rank_loads_at_phase = this->info_.getAllRankLoadsAtPhase(phase);
-    for (auto [rank, rank_load] : rank_loads_at_phase){
-      rq_sum += rank_load;
-    }
-    return rq_sum / rank_loads_at_phase.size();
-  }
-  else{
-    throw std::runtime_error("Invalid QOI: " + qoi);
+  auto rank_loads_at_phase = this->info_.getAllRankQOIAtPhase(phase, qoi);
```

Comment:
Does this need to be const& to avoid copies? Not sure of the signature

Reply:
Fixed